packages/create-mc-app/src/steps/success.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+const { shouldUseYarn } = require('../utils');
+
+module.exports = function success({ projectDirectoryName, templateName }) {
+  console.log();
+  console.log(`Success! 🎉`);
+  console.log(
+    `The Merchant Center application ${projectDirectoryName} was successfully bootstrapped based on the ${templateName} template.`
+  );
+  console.log();
+  console.log(`To get started:`);
+  console.log(`  $ cd ${projectDirectoryName}`);
+  console.log(`  $ ${shouldUseYarn ? 'yarn' : 'npm'} start`);
+  console.log();
+  console.log(`For more info, have a look at the README. Enjoy 🚀`);
```

Comment:
Nice effort to get the output looking sharp. I always wonder if something like this is fun to play with e.g. signale
https://github.com/klaussinani/signale.

Reply:
Yeah I also wanted to look into something like this. I didn't know that library, I'll check it out thanks! 💯